src/index.js

```diff
@@ -18,18 +18,27 @@ const nanoid = (size = 21) => {
 }
 
  // Create script to init hCaptcha
-const CaptchaScript = (cb, hl) => {
-    let script = document.createElement("script")
-
-    window.hcaptchaOnLoad = cb;
+let onLoadListeners = [];
+let captchaScriptCreated = false;
+
+// Create global onload callback
+window.hcaptchaOnLoad = () => {
+  // Iterate over onload listeners, call each listener
+  onLoadListeners = onLoadListeners.filter(listener => {
+    listener();
+    return false;
+  });
+};
```

Comment:
Any chance it would break SSR? Previously `window` was only accessed in `componentDidMount`

Reply:
Good catch, since SSR does not call `componentDidMount, I moved the window element back into the `CaptchaScript` function.